packages/core/src/media/metrics/metrics.ts

```diff
@@ -319,11 +319,11 @@ export class MetricsStatus<TElement, TMediaStream> {
   getMetrics() {
     const currentMetrics: RawMetrics = {
       ...this.currentMetrics,
-      playerHeight: this.store.getState().size?.container?.height ?? null,
-      playerWidth: this.store.getState().size?.container?.width ?? null,
-      videoWidth: this.store.getState().size?.media?.width ?? null,
-      videoHeight: this.store.getState().size?.media?.height ?? null,
-      duration: this.store.getState().duration,
+      playerHeight: this.store.getState().size?.container?.height || null,
```

Comment:
The idea here is to convert any `0` to `null` as well, is that right?

Reply:
Yeah exactly - if it's zero we shouldn't report it